src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -55,6 +55,8 @@ public class FileNameBuilder : IBuildFileNames
         public static readonly Regex SeasonRegex = new Regex(@"(?<season>\{season(?:\:0+)?})",
                                                               RegexOptions.Compiled | RegexOptions.IgnoreCase);
 
+        public static readonly Regex MultiRegex = new (@"[_. ](?<multi>multi)[_. ]", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
This should go in `Parser`, since it is for parsing, not renaming files.

Reply:
Ok, I've moved it to `Parser` and add a new function `IsMultiLanguages` as all other RegEx were private.